src/components/textarea.js

```diff
@@ -48,6 +48,9 @@ module.exports = function(app) {
           if ($scope.component.wysiwyg) {
             return 'formio/componentsView/content.html';
           }
+          else if ($scope.submissionViewMode) {
```

Comment:
Not sure this is the correct solution but is close. @travist Is there ever a time we would want to display the textarea component in the view rendering of textareas? I think we would always want to render componentsView/content.html here. That way we wouldn't need the new submissionViewMode property.